tests/queries/0_stateless/03444_distributed_wo_hedged_requests.sql

```diff
@@ -0,0 +1,26 @@
+drop table if exists 03444_local;
+drop table if exists 03444_distr;
+
+create table 03444_local (id UInt32) engine = MergeTree order by id as select * from numbers(100);
+create table 03444_distr (id UInt32) engine = Distributed(test_cluster_one_shard_two_replicas, currentDatabase(), 03444_local);
+
+select 'remote() 1 shard: ' || count()
+from remote('127.0.0.1|127.0.0.2|127.0.0.3', currentDatabase(), 03444_local)
+settings prefer_localhost_replica = 0,
+         use_hedged_requests = 0,
```

Comment:
Let's also test with and without use_hedged_requests and with and without async_socket_for_remote